common/openssl-compat.h

```diff
@@ -10,8 +10,11 @@
 #ifndef LIBCRYPTO_COMPAT_H
 #define LIBCRYPTO_COMPAT_H
 
+extern char openssl_compat_used;
+
 #include <openssl/opensslv.h>
-#if OPENSSL_VERSION_NUMBER < 0x10100000L
+#if (OPENSSL_VERSION_NUMBER < 0x10100000L) ||                                  \
+  (OPENSSL_VERSION_NUMBER == 0x20000000L)
```

Comment:
should we maybe use a libressl specific define instead to detect libressl?

Reply:
Fixed in 92e42d94ae5ca6e994e9cce664ae91fd12c8f9ee